src/process_utility.c

```diff
@@ -183,13 +170,24 @@ relation_not_only(RangeVar *rv)
 				 errmsg("ONLY option not supported on hypertable operations")));
 }
 
+static void
+process_add_hypertable(ProcessUtilityArgs *args, Hypertable *ht)
+{
+	args->hypertable_list = lappend(args->hypertable_list, ht);
+}
+
+static void
+process_add_subcmd(ProcessUtilityArgs *args, void *datum)
```

Comment:
I vould avoid calling the parameter `datum` here as there is a specific `Datum` type in PostgreSQL and the argument here is AFAIK typically not a datum. If not `void`, I believe the correct type would be `Node *`, since all commands should be a node type.